include/seastar/http/internal/content_source.hh

```diff
@@ -40,15 +40,17 @@ namespace internal {
  * */
 class content_length_source_impl : public data_source_impl {
     input_stream<char>& _inp;
-    size_t _remaining_bytes = 0;
-    size_t* _consumed_bytes;
+    size_t _builtin_remaining_bytes = 0;
```

Comment:
It's not possible to understand, or even guess, how _builtin_remaining_bytes differ from _remaining_bytes, what "builtin" refers to. Please either reconsider the names, or add a comment.

Reply:
This variable is needed to make `size_t& _remaining_bytes` reference something in case the class is constucted without "external" remaining bytes counter

Reply:
It doesn't help that you write this answer in a review comment, not a comment. How will anybody who reads this in the future understand?